Задание 5/Задание 5/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+       
+        double x = CalculateX(4);
+
+        
+        Console.WriteLine($"Значение x: {x:F3}");
+    }
+
+   
+    static double CalculateX(int n)
+    {
+        
+        double result = n;
```

Comment:
Метод написан неверно. Вот листинг
```
Значение x: 0,994
```
На самом деле значение равно 0,976069396758539. Близко, но не совпадает.
Ошибка здесь:
```suggestion
        double result = Math.SIn(n);
```